versioned_docs/version-8.6/self-managed/setup/deploy/local/c8run.md

```diff
@@ -25,8 +25,8 @@ Camunda 8 Run includes the following:
 
 ## Prerequisites
 
-:::warning
-Due to a bug in 8.6, cookie authentication is required to use the Tasklist, Operate, and Zeebe APIs included in Camunda 8 Run. For an example of authenticating via cookie, see the [Operate API documentation](/apis-tools/operate-api//authentication.md#authentication-via-cookie-self-managed-only).
+:::caution
```

Comment:
Being honest, to me this does not make sense. This installation guide does not reference the Camunda REST API at all. The page still focuses on the old architecture by talking about Zeebe configuration, gRPC endpoints, etc.